modules/apps/frontend-token/frontend-token-definition-test/src/testIntegration/java/com/liferay/frontend/token/definition/test/FrontendTokenDefinitionRegistryTest.java

```diff
@@ -184,11 +209,24 @@ private void _assertFrontendTokenDefinition(
 					"JSONObject/0")),
 			String.valueOf(
 				frontendTokenMapping.getJSONObject(LocaleUtil.ENGLISH)));
+
+		Assert.assertEquals(
+			frontendTokenDefinitionName, frontendTokenDefinition.getThemeId());
 	}
 
+	private static final String _THEME_ID_CLIENT_EXTENSION_ENTRY =
+		RandomTestUtil.randomString();
+
+	private static final String _THEME_ID_LAYOUT_SET =
+		"testfrontendtokendefinition";
```

Comment:
Hi @algobob I know that Brian renamed this but I believe this should be renamed to `_CLIENT_EXTENSION_ENTRY_THEME_ID` and `_LAYOUT_SET_THEME_ID`, lets rename this and we can talk to Brian together if he disagree, I'm following [this pattern](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/wiki/wiki-test/src/testIntegration/java/com/liferay/wiki/trash/test/WikiPageDependentsTrashHandlerTest.java#L1066-L1068).

Also the value in my opinion should be:

````java
	private static final String _CLIENT_EXTENSION_ENTRY_THEME_ID =
		RandomTestUtil.randomString();

	private static final String _LAYOUT_SET_THEME_ID =
		RandomTestUtil.randomString();
````

if the theme id cannot be random I believe it should be renamed to `testLayoutSetThemeId` or `TEST_LAYOUT_SET_THEME_ID`

Thank you!

Reply:
cool. the second one cant be random, so I'll rename to `_TEST_LAYOUT_SET_THEME_ID`

Reply:
@algobob why can't `_TEST_LAYOUT_SET_THEME_ID` be random?

Also if it really can't be random we usually want the value and the name to more directly correspond... something like:
```
private static final String _TEST_LAYOUT_SET_THEME_ID =
	"testLayoutSetThemeId";
```

Reply:
@ethib137  It can't be random because it refers to a deployed theme [here](https://github.com/liferay-platform-experience/liferay-portal/blob/7174aa8dc2c97bed6b76afd0763a83a59a1c7f7b/modules/apps/frontend-token/frontend-token-definition-test/src/testIntegration/resources/WEB-INF/liferay-look-and-feel.xml#L8).

Also, that id was set by Brian 7 months ago when he was reviewing my PR

Reply:
@MarinhoFeliphe we can't change the values neither. During theme processing, the hifens are removed.